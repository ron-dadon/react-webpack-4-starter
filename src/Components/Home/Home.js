import React from 'react'

const Home = () => <h1>React Webpack 4 Starter</h1>

export default Home
