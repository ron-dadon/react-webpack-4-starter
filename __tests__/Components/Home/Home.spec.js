import Home from 'Components/Home/Home'
import { shallowBasicTest } from 'testUtils/test-template'

shallowBasicTest({ Component: Home })
