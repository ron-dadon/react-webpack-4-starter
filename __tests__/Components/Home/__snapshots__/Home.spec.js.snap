// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Should have props 1`] = `
Object {
  "children": "React Webpack 4 Starter",
}
`;

exports[`Home Should have structure 1`] = `
<h1>
  React Webpack 4 Starter
</h1>
`;

exports[`Home Should render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "React Webpack 4 Starter",
    },
    "ref": null,
    "rendered": "React Webpack 4 Starter",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "React Webpack 4 Starter",
      },
      "ref": null,
      "rendered": "React Webpack 4 Starter",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
